hook.js

```diff
@@ -16,33 +16,38 @@ var githubClient = github.client();
 function handle(request, response) {
     var eventType = request.headers['x-github-event'];
     var metadata = request.body;
+    var originRepo;
 
     response.status(200).send('OK');
 
     // React to pull requests only
     if (eventType === 'pull_request') {
+        originRepo = metadata.pull_request.head.repo.full_name;
+
         logger.log(metadata);
-        comment(metadata);
+        trackUsage(originRepo);
+
+        addPullRequestComments(
+            metadata.repository.full_name,
+            originRepo,
+            metadata.pull_request.head.ref,
+            metadata.number,
+            metadata.pull_request.review_comments_url
+        );
     }
 }
 
-function comment(metadata) {
-    var prNumber = metadata.number;
-    var destinationRepo = metadata.repository.full_name;
-    var originRepo = metadata.pull_request.head.repo.full_name;
-    var originBranch = metadata.pull_request.head.ref;
-
-    trackUsage(originRepo);
-
+// TODO: Remove the commentURL parameter once parseCSS is refactored
```

Comment:
BTW, did anyone else notice that this line auto-magically triggered [a corresponding GitHub issue](https://github.com/mdn/discord/issues/75) via [another of @mrmakeit's hooks](https://github.com/mrmakeit/github-todo).

#magic-makes-magic

![giphy](https://cloud.githubusercontent.com/assets/71928/8415177/8f077abe-1e62-11e5-9c7a-f96346462f67.gif)